sdks/python/test-suites/tox/py37/build.gradle

```diff
@@ -40,6 +40,10 @@ test.dependsOn testPy37Gcp
 
 toxTask "testPy37Cython", "py37-cython"
 test.dependsOn testPy37Cython
+
+toxTask "testPy37Interactive", "py37-interactive"
```

Comment:
Maybe just limit tests to py 3.7 to limit the added initial test time.

Reply:
Got it, I've removed py 3.6 configuration and tasks.